### ISSUE TITLE
Implement Object.defineProperty instead of Bridge.get

### ISSUE THREAD

**Issue by geoffreymcgill** [bridgedotnet/Bridge#1050]
Once implemented, post a follow-up in [this forum thread](http://forums.bridge.net/forum/bridge-net-pro/bugs/1871).

### Expected

```javascript
Demo.App.something = 1;
```

### Actual

```javascript
Bridge.get(Demo.App).something = 1;
```

### Steps To Reproduce

```csharp
public class App
{
    static int something;
    
    [Ready]
    public static void Main()
    {
        App.something = 1;
    }
}
```

Live Bridge sample:
http://live.bridge.net/#b2f59e0776a60a15946e

**Comment by geoffreymcgill** [comment]
IE8 support might be dropped because of its poor implementation of `Object.defineProperty` and an acceptable shim may not be available.

_Possible_ work-arounds include:

1. https://github.com/inexorabletash/polyfill/blob/master/es5.js#L71

**Comment by geoffreymcgill** [comment]
If IE8 is dropped, we need to update the list of supported browsers:
http://bridge.net/download/#release-notes

Also, Microsoft has dropped support of IE8 as of Jan 12, 2016. Actually they dropped support for IE8, IE9 and IE10 all at once. More:
https://www.microsoft.com/en-ca/WindowsForBusiness/End-of-IE-support

jQuery 2 only supports IE9+

**Comment by LMLB** [comment]
Actually, Microsoft dropped support for all versions of IE except whatever version is the latest on each supported version of Windows. See: https://support.microsoft.com/gp/Microsoft-Internet-Explorer

**Comment by DaniilVeriga** [comment]
I'd like to add that possible drop of IE8 support is going to unleash improvements to be done in Bridge getting rid of IE8 support hacks and also releasing resources to be used for new features development.

**Comment by SamuelEnglard** [comment]
I think this is a good idea, I would release a final "IE8 compatible" version with all the fixes that you can.

**Comment by DaniilVeriga** [comment]
Hi @SamuelEnglard,

Thank you for the feedback. We have decided to drop IE8 support - #1115.

So, let's consider 1.11 as a final IE8 compatible version.

**Comment by geoffreymcgill** [comment]
The `Object.defineProperty` functionality has been implemented in Bridge 16.0. Please see Issue #1229.
